src/sentry/api/endpoints/organization_eventid.py

```diff
@@ -47,33 +47,22 @@ def get(self, request, organization, event_id):
                 'id', 'slug'))
 
         try:
-            event = Event.objects.filter(event_id=event_id,
-                                         project_id__in=project_slugs_by_id.keys())[0]
+            event = eventstore.get_events(filter_keys={
```

Comment:
I'd wait the other method you introduced in https://github.com/getsentry/sentry/pull/14249 instead of abusing get_events. Is there any other place where you could showcase get_events ?

Reply:
The other method is for when you have both a project ID and event ID. This scenario is a pretty rare case where we have an event ID but want to search across a number of projects, and might have more than one result returned from the eventstore.